plugins/c8y_configuration_plugin/src/tests.rs

```diff
@@ -628,6 +628,20 @@ async fn test_handle_config_update_request_tedge_device() -> anyhow::Result<()>
         .with(predicate::always())
         .returning(|_path| false);
```

Comment:
I don't think this `url_is_in_my_tenant_domain()` API mock is required anymore either, as we're directly mocking the `download_file` API.

Reply:
Also true!